cmd/ocsp-updater/main.go

```diff
@@ -196,19 +196,15 @@ func generateOCSPCacheKeys(req []byte, ocspServer string) []string {
 	}
 }
 
-// sendPurge should only be called as a Goroutine as it will block until the purge
-// request is successful
-func (updater *OCSPUpdater) sendPurge(der []byte) {
+func (updater *OCSPUpdater) generatePurgeURLs(der []byte) ([]string, error) {
 	cert, err := x509.ParseCertificate(der)
 	if err != nil {
-		updater.log.AuditErrf("Failed to parse certificate for cache purge: %s", err)
```

Comment:
This is a fairly serious error that should never happen; can you restore the log line?

Reply:
It still gets logged, just as the callsite for `generateRevokedResponse`. Keeping this log line just makes it get logged twice.